exporter/sentryexporter/sentry_exporter.go

```diff
@@ -480,7 +480,8 @@ func CreateSentryExporter(config *Config, set exporter.CreateSettings) (exporter
 	transport := newSentryTransport()
 
 	clientOptions := sentry.ClientOptions{
-		Dsn: config.DSN,
+		Dsn: 		 config.DSN,
+		Environment: config.ENV,
```

Comment:
The problem setting config here is that it globally applies to every single event coming through the collector, regardless of the state of all the services sending requests to the collector. Is that what we would want?

Reply:
Discussed in https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/18694 - we can leave this for now.